backend/prisma/schema.prisma

```diff
@@ -0,0 +1,37 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id        Int     @id @default(autoincrement())
+  email     String  @unique
+  username  String
+  password  String
+  posts     Post[]
+  skills    Skill[]
+}
+
+model Post {
```

Comment:
Sorry if i missed this but what will we be using this table for? Are users making skill swap posts on the app? More clarification would be great!